src/IO/Radio.h

```diff
@@ -13,15 +13,15 @@
 class Radio : public IO
 {
 public:
-	Radio(EventQueue &eventQueue);
+	explicit Radio(EventQueue &eventQueue);
```

Comment:
Why is only this constructor `explicit`?

Reply:
I was reading it is to prevent implicit conversion when doing operations. See https://stackoverflow.com/questions/121162/what-does-the-explicit-keyword-mean. This only applies to constructors having only 1 argument. 

Is it needed? No, but I let it do this change because it's technically what we want here.